docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -218,6 +369,21 @@ Alias: `when`
 
 Clears the _context_, and returns `null`.
 
+*Expression syntax*
+[source,js]
+----
+clear
+----
+
+*Code example*
+[source,text]
+----
+filters 
+| demodata
+| clear
+| render
+----
```

Comment:
Can we remove this example? It's not very useful, and I'd like to come up with a better one.